### PR TITLE
[12.0][DOC] account_asset replaced by OCA

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Fine tunning of https://github.com/OCA/OpenUpgrade/commit/4f1b58559ef590cca00a28adc5dc23aada0bd7be, that adds the documentation for `account_asset`.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/openupgrade/doc/source/modules110-120.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ missing in the new release are marked with |del|.
 +--------------------------------------------+-------------------------------------------------+
 |account_analytic_default                    |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-| |del| account_asset                        |                                                 |
+| |del| account_asset                        | Moved to OCA [#account_asset]_                  |
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules110-120.rst]: s/Moved to OCA/Replaced by OCA module

Comment (MiquelRForgeFlow) [reply odoo/openupgrade/doc/source/modules110-120.rst]: done

File: odoo/openupgrade/doc/source/modules110-120.rst [hunk 2]
```diff
@@ -27,7 +27,7 @@ missing in the new release are marked with |del|.
 +--------------------------------------------+-------------------------------------------------+
 |account_analytic_default                    |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-| |del| account_asset                        |                                                 |
+| |del| account_asset                        | Replaced by OCA [#account_asset]_               |
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules110-120.rst]: s/OCA/OCA module, and even better to put directly `account_asset_management` in the link name

File: odoo/openupgrade/doc/source/modules110-120.rst [hunk 3]
```diff
@@ -638,6 +638,10 @@ missing in the new release are marked with |del|.
 |website_twitter                             |                                                 |
 +--------------------------------------------+-------------------------------------------------+
 
+.. [#account_asset] 'Account Asset' Module is now under Odoo Community Association Umbrella
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules110-120.rst]: I would rephrase this to say that it's alternative module. Right now, it seems like other forward-portings done like `crm_claim`.

File: odoo/openupgrade/doc/source/modules110-120.rst (hunk 2 above)
Comment (MiquelRForgeFlow) [reply odoo/openupgrade/doc/source/modules110-120.rst]: the link name it's not visible, so I prefer the shorter one (the longer one doesn't fit)

File: odoo/openupgrade/doc/source/modules110-120.rst [hunk 4]
```diff
@@ -638,6 +638,10 @@ missing in the new release are marked with |del|.
 |website_twitter                             |                                                 |
 +--------------------------------------------+-------------------------------------------------+
 
+.. [#account_asset] 'Account Asset' module it is replaced by the Odoo Community Association module
```
Comment (pedrobaeza) [inline odoo/openupgrade/doc/source/modules110-120.rst]: s/it is/is (the subject is already there)

Comment (MiquelRForgeFlow) [reply odoo/openupgrade/doc/source/modules110-120.rst]: yeah, it's a typo
